src/plugins/core/configuration/plugin-configuration.ts

```diff
@@ -125,7 +125,13 @@ export class PluginConfigurationData {
 	}
 
 	logo() {
-		const logo: string | undefined = this.#config.get<string>("logo");
+		let logo: string | undefined;
+
+		if (this.#config.has("logo")) {
+			logo = this.#config.get("logo");
+		} else {
+			logo = this.#manifest.get("logo");
```

Comment:
In which scenario would this be reached?

Reply:
The first line is for npm packages (received from sdk), as we create them as `PluginConfigurationData.make({ logo: config.logo() })`.

Then the "desktop-wallet" field is used to specify the logo (packages loaded from the folder): https://github.com/ark-ecosystem-desktop-plugins/sound-notifications/blob/master/package.json#L9